app/models/institution.rb

```diff
@@ -0,0 +1,180 @@
+# frozen_string_literal: true
+
+class Institution < ActiveRecord::Base
+  include Tristateable
+
+  EMPLOYER = 'ojt'
+  LOCALE = {
+    11 => 'City',
+    12 => 'City',
+    13 => 'City',
+    21 => 'Suburban',
+    22 => 'Suburban',
+    23 => 'Suburban',
+    31 => 'Town',
+    32 => 'Town',
+    33 => 'Town',
+    41 => 'Rural',
+    42 => 'Rural',
+    43 => 'Rural'
+  }
+  DEGREES = {
+    0 => nil,
+    "ncd" => "Certificate",
+    1 => "Certificate",
+    "2-year" => 2,
+    2 => 2,
+    3 => 4,
+    4 => 4,
+    "4-year" => 4
```

Comment:
Another nit-pick (sorry!) 
Double quoting will cause Rubocop to freak-out, the guidelines call for single quotes unless interpolating.